changelog/8.6.0.yaml

```
@@ -0,0 +1,212 @@
+      file:
+        name: 1668483030-common-expression-language.yaml
+        checksum: 82427fc514372c067e140837177c8d81131ed424
+    - kind: feature
+      summary: Capture stdout/stderr of all subprocesses and adjust default log level to info for all components.
+      description: ""
+      component: ""
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1702
+      issue:
+        - https://github.com/elastic/elastic-agent/issues/221
+      timestamp: 1669236059
+      file:
+        name: 1669236059-Capture-stdout-stderr-of-all-spawned-components-to-simplify-logging.yaml
+        checksum: b4cf9cff0ef13bd168cadc80f89c4969b49fb378
+    - kind: bug-fix
+      summary: Elastic-agent will use local port when running fleet-server
+      description: The elastic-agent will now use the 8221 locally bound port when running fleet-server instead of the external port (8220).
+      component: ""
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1867
+      issue:
+        - https://github.com/elastic/ingest-dev/issues/1394
+      timestamp: 1669929902
+      file:
+        name: 1669929902-fix-local-fleet-server-port.yaml
+        checksum: 94f37fd1ff74bd320ec25cfcf2e2a0890eda494a
+    - kind: feature
+      summary: Remove inputs when all streams are removed
+      description: ""
+      component: ""
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1869
+      issue:
+        - https://github.com/elastic/elastic-agent/issues/1868
+      timestamp: 1669940990
+      file:
+        name: 1669940990-Remove-inputs-when-all-streams-are-removed.yaml
+        checksum: 3fcf26a5bbb988db85c0c5923d4bd2284dc95866
+    - kind: feature
+      summary: No longer restart Elastic Agent on log level change
+      description: ""
+      component: ""
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1914
+      issue:
+        - https://github.com/elastic/elastic-agent/issues/1896
+      timestamp: 1670518841
+      file:
+        name: 1670518841-No-longer-restart-Elastic-Agent-on-log-level-change.yaml
+        checksum: 12b34904cd47182a45e6e113518ef91ec718ebac
+    - kind: breaking-change
+      summary: Each input in an agent policy must have a unique ID.
+      description: |
+        Each input in an agent policy must have a unique ID, like "id: my-unique-input-id".
+        This only affects standalone agents. Unique IDs are automatically generated in
+        agent policies managed by Fleet.
+      component: agent
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1994
+      issue: []
+      timestamp: 1671748292
+      file:
+        name: 1671748292-input-ids-are-required.yaml
+        checksum: 3ca6d7c10ce16c72c74b67b18916407ad09459fd
+    - kind: upgrade
+      summary: "New control protocol between the Elastic Agent and its subprocesses. \nEnables per integration health reporting and simplifies new input development, among other things.\n"
```

```suggestion
      summary: |
            New control protocol between the Elastic Agent and its subprocesses. 
            Enables per integration health reporting and simplifies new input development,             among other things
```

Probably don't want the literal `\n` characters in the text. Beware the indentation here might not be correct, I can't tell if it is using tabs or spaces in the suggestion box on GitHub.